project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/vendor/aos/aos.esm.js

```
@@ -0,0 +1,610 @@
+};
+
+/**
+ * Set or remove aos-animate class
+ * @param {node} el         element
+ * @param {int}  top        scrolled distance
+ */
+var applyClasses = function applyClasses(el, top) {
+  var options = el.options,
+      position = el.position,
+      node = el.node,
+      data = el.data;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)